index.js

```diff
@@ -396,28 +427,42 @@ SendStream.prototype.isRangeFresh = function isRangeFresh(){
  * @private
  */
 
-SendStream.prototype.redirect = function redirect(path) {
+SendStream.prototype.redirectDirectory = function redirectDirectory(path) {
```

Comment:
Changing this public method name is a breaking change, meaning accepting this would require a major version bump. There is no timeline for when a major will happen due to Express support, if that is OK.